### DISCUSSION TITLE
Accurate parameter from CursorOnObject have no effect

### DISCUSSION THREAD

**Discussion by Bouh** [4ian/GDevelop#3641] [Enhancement ideas (and their technical discussions)]
## Describe the bug
The condition CursorOnObject[ have a parameter](https://github.com/4ian/GDevelop/blob/master/GDJS/Runtime/events-tools/inputtools.ts#L264) `accurate` not used in the engine (GD5).
This is a remnant from GD4. This boolean was used for checking if the cursor is on a sprite with a pixel perfect detection, all transparent pixels are ignored.
The C++ code with SFML still exist, you can see [the previous implementation.](https://github.com/4ian/GDevelop/blob/master/GDCpp/GDCpp/Runtime/RuntimeSpriteObject.cpp#L531)

## Other details
* Beta 110

**Comment by 4ian** [comment]
We might consider to drop this parameter entirely for now?

**Comment by Bouh** [comment]
There is something possible with [hitarea](https://pixijs.download/dev/docs/PIXI.Sprite.html#hitArea) but it's limited and require to create a convex hull.
I've no idea for checking the transparency of sprites.

So yeah this parameter can be removed or hidden.

Complaint about this missing feature:
https://forum.gdevelop-app.com/t/hitboxes-valables-pour-le-curseur/33566
